addons/account_peppol/models/account_edi_proxy_user.py

```diff
@@ -583,3 +588,21 @@ def _get_user_from_token(self, token: str, url: str):
     def _peppol_reset_webhook(self):
         for edi_user in self:
             edi_user._call_peppol_proxy('/api/peppol/2/set_webhook', params={'webhook_url': edi_user.company_id._get_peppol_webhook_endpoint(), 'token': edi_user._generate_webhook_token()})
+
+    def _peppol_update_metadata(self):
+        self.ensure_one()
+        try:
+            features_flags = self._call_peppol_proxy(
+                '/api/peppol/1/get_features',
+                params={
+                    'available_features': list(self.company_id._peppol_get_default_feature_flags().keys())
+                }
+            )
+        except AccountEdiProxyError:
+            features_flags = {}  # fallback to default if error occurs
```

Comment:
```suggestion
            features = {}  # fallback to default if error occurs
```
Naming tends to suggest a set while it's a dict